### PR TITLE
EWPP-355: Update field definitions when updating vocabularies

### PR DESCRIPTION
**Pull request by brummbar**

### CODE REVIEW

**Review by imanoleguskiza — DISMISSED** [review]

File: src/OpenVocabularyStorage.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Constructs a OpenVocabularyStorage object.
```
Comment (nagyad) [inline src/OpenVocabularyStorage.php]: Constructs an OpenVocabularyStorage object.

File: src/OpenVocabularyStorage.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      $container->get('entity_type.manager'),
+      $container->get('entity_field.manager'),
+      $container->get('entity.memory_cache')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function doPostSave(EntityInterface $entity, $update) {
```
Comment (nagyad) [inline src/OpenVocabularyStorage.php:80]: We don't need type hint for $update?

**Review by nagyad — CHANGES_REQUESTED** [review]

File: src/OpenVocabularyStorage.php (hunk 2 above)
Comment (brummbar) [reply src/OpenVocabularyStorage.php:80]: You tell me, isn't it an overridden method and thus signature shouldn't be changed? :)

File: src/OpenVocabularyStorage.php (hunk 1 above)
Comment (brummbar) [reply src/OpenVocabularyStorage.php]: Opsie!

**Review by nagyad — APPROVED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Pull request merged**